src/xccl/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# XCCL sources
+
+file(GLOB xccl_h "*.hpp")
+file(GLOB xccl_cpp "*.cpp")
+
+list(APPEND ATen_XPU_XCCL_SRCS ${xccl_cpp})
+
+set(ATen_XPU_XCCL_SRCS ${ATen_XPU_XCCL_SRCS} PARENT_SCOPE)
+
+foreach(HEADER  ${xccl_h})
```

Comment:
Please add comment for this header handling.

Reply:
Sure

Reply:
Pls. give more detailed comments there.
1. why we have to copy the header to below path?
2. when this header will be used? Build which component.
3. future plan.